package.json

```diff
@@ -11,11 +11,9 @@
     "docs:config": "./scripts/create-library-config.js",
     "docs:prod": "npm run docs:compile && cd docs && bundle install && bundle exec jekyll build --config _config.yml,_config-library.yml,_config-prod.yml && cd ..",
     "docs": "npm run docs:compile && cd docs && bundle install && bundle exec jekyll build --config _config.yml,_config-library.yml && cd ..",
-    "docs:lint": "stylelint './docs/_sass/*.scss'",
```

Comment:
Just deleting these entirely because I imagine the future will have a single lint script for the entire repo.